tensorflow/python/data/experimental/kernel_tests/service/cross_trainer_cache_test.py

```diff
@@ -71,8 +73,9 @@ def testDisableCrossTrainerCacheByDefault(self):
       combinations.times(
           combinations.combine(tf_api_version=2, mode=["eager", "graph"])))
   def testConcurrentReaders(self):
+    num_cpus = multiprocessing.cpu_count()
     cluster = self._create_cluster(
-        num_workers=1, cross_trainer_cache_size_bytes=18000)
+        num_workers=1, cross_trainer_cache_size_bytes=(num_cpus + 8) * 363)
```

Comment:
Could you add a comment for what this means? Thank you.

Reply:
Comment added.